### PR TITLE
changes to README and script files

### PR DESCRIPTION
**Pull request by sreesindhusruthiyadavalli**

### CODE REVIEW

File: tf-mnist/README.md [hunk 1]
```diff
@@ -149,17 +150,21 @@ Now try a different image in `data` directory :)
...
 This is ideal if you would like to create a test web application exposed by a loadbalancer.
+    source ./variables.bash
 
     MNIST_SERVING_IP=`kubectl -n ${NAMESPACE} get svc/mnist --output=jsonpath={.spec.clusterIP}`
     echo "MNIST_SERVING_IP is ${MNIST_SERVING_IP}"
 
 Create image using Dockerfile in the webapp folder and upload to DockerHub
 
-    CLIENT_IMAGE=${DOCKER_BASE_URL}/mnist-client
+    #CLIENT_IMAGE=${DOCKER_BASE_URL}/mnist-client- this can't be accessed so use the below image
```
Comment (amsaha) [inline tf-mnist/README.md]: This line should not be commented out and johnugeorge specific information should not be put here. I think we should raise an issue that needs to be fixed.

File: tf-mnist/README.md [hunk 2]
```diff
@@ -169,9 +174,9 @@ Create image using Dockerfile in the webapp folder and upload to DockerHub
 
 Now get the loadbalancer IP of the tf-mnist-client service
 
-    kubectl get svc/tf-mnist-client -n ${NAMESPACE} -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+    kubectl get svc/tf-mnist-client -n ${NAMESPACE} -o jsonpath='{.status.loadBalancer.ingress[0].ip}' - outputs the IP
```
Comment (amsaha) [inline tf-mnist/README.md]: This is not needed. The comment above already says that this gives the loadbalancer IP.

File: tf-mnist/README.md [hunk 3]
```diff
@@ -169,9 +174,9 @@ Create image using Dockerfile in the webapp folder and upload to DockerHub
...
 
-Open browser and see app at http://LoadBalancerIP
+Open browser and see app at http://LoadBalancerIP
```
Comment (amsaha) [inline tf-mnist/README.md]: please delete whitespace at the end.

File: tf-mnist/cleanup.bash [hunk 4]
```diff
@@ -7,7 +7,7 @@ pwd
 ks delete ${KF_ENV} -c tfserving
 kubectl get pods -n ${NAMESPACE}
 
-JOB=tf-${APP_NAME}job
+JOB=tf-mnistjob
```
[the remark this answers is not in the document]
Comment (sreesindhusruthiyadavalli) [reply tf-mnist/cleanup.bash]: here APP_NAME and job name are two different things. For this example app_name and jobname happened to be same.

[the remark this answers is not in the document]
Comment (amsaha) [reply tf-mnist/cleanup.bash]: In that case, please add a new variable to variables.bash

File: tf-mnist/portf.bash [hunk 5]
```diff
@@ -14,6 +14,8 @@
 # Read variables
 source variables.bash
 
+#Get the labels for the serving pod
+#kubectl -n ${NAMESPACE} get pod --show-labels
```
[the remark this answers is not in the document]
Comment (sreesindhusruthiyadavalli) [reply tf-mnist/portf.bash]: Updated this

File: tf-mnist/serve.bash [hunk 6]
```diff
@@ -19,7 +19,7 @@ pwd
 
 #2. Create namespace if not present
 
-ks generate tf-serving tfserving --name=${APP_NAME}
+ks generate tf-serving tfserving --name="mnist"
```
[the remark this answers is not in the document]
Comment (sreesindhusruthiyadavalli) [reply tf-mnist/serve.bash]: changing app_name variable changes the model name which needs a code change in model_client.py at the below line:
request.model_spec.name = "mnist"

File: tf-mnist/train.bash [hunk 7]
```diff
@@ -17,7 +17,7 @@ pwd
 # training job when created.
 ENV="TF_DATA_DIR=$TF_DATA_DIR,TF_EXPORT_DIR=$TF_EXPORT_DIR,TF_MODEL_DIR=$TF_MODEL_DIR"
 
-JOB=tf-${APP_NAME}job
+JOB=tf-mnistjob
```
[the remark this answers is not in the document]
Comment (sreesindhusruthiyadavalli) [reply tf-mnist/train.bash]: job name is different from app_name incase we change app_name to other than "mnist" this doesn't work.

[the remark this answers is not in the document]
Comment (amsaha) [reply tf-mnist/train.bash]: As explained earlier, please introduce a new variable in variables.bash and use that in the scripts.

File: tf-mnist/cleanup.bash (hunk 4 above)
[the remark this answers is not in the document]
Comment (sreesindhusruthiyadavalli) [reply tf-mnist/cleanup.bash]: we can add a variable JOB_NAME=tf-mnistjob

File: tf-mnist/serve.bash (hunk 6 above)
[the remark this answers is not in the document]
Comment (ramdootp) [reply tf-mnist/serve.bash]: you may parameterize the mnist_client.py to take APP_NAME

File: tf-mnist/README.md [hunk 8]
```diff
@@ -149,17 +150,20 @@ Now try a different image in `data` directory :)
...
 
 Create image using Dockerfile in the webapp folder and upload to DockerHub
 
     CLIENT_IMAGE=${DOCKER_BASE_URL}/mnist-client
+    ---use this if you have done any code changes, otherwise you can skip the following two docker commands--	
     docker build . --no-cache  -f Dockerfile -t ${CLIENT_IMAGE}
     docker push ${CLIENT_IMAGE}
 
     echo "CLIENT_IMAGE is ${CLIENT_IMAGE}"
+    cd mnist
```
Comment (ramdootp) [inline tf-mnist/README.md]: why hardcode here?

**Pull request closed without merging**